test/run_tests.sh

```diff
@@ -139,6 +139,7 @@ function run_mp_op_tests {
   run_xla_backend_mp python3 "$CDIR/test_torch_distributed_all_reduce_xla_backend.py"
   run_xla_backend_mp python3 "$CDIR/test_torch_distributed_multi_all_reduce_xla_backend.py"
   run_xla_backend_mp python3 "$CDIR/test_torch_distributed_reduce_scatter_xla_backend.py"
+  run_xla_backend_mp python3 "$CDIR/test_ddp.py"
```

Comment:
I guess on CPU this test is not doing anything since by default we have 1 CPU, but on GPU we can test it on 2 gpu at least.

Reply:
Hmmm, why are we running these tests in cpu env then? I saw the tests do early exit if the device is CPU...